### PR TITLE
Redux test

### PR DESCRIPTION
**Pull request by petermason**
# Description

Add example redux test

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [ ] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

image

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevent
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: src/core/state/app/slice/__tests__/index.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import store from '../../../store';
+import { initialState, reset, setDashboardHasBeenViewed, setLoggedVaccine, setMentalHealthStudyActive } from '../index';
+// import { deleteBook, updateBook, addNewBook } from '../redux/bookSlice';
```
Comment (selvach) [inline src/core/state/app/slice/__tests__/index.ts:3]: Could you remove this comment important, please.

**Review by selvach — DISMISSED** [review]
Yay... tests.  I guess it's a good start.

**Review by bionara — DISMISSED** [review]
very nice! I like how concise each case is - have seen some horrific state test cases with MASSIVE json blobs!

**Review by selvach — APPROVED** [review]

**Review by bionara — APPROVED** [review]
+1

**Pull request merged**